benchmarks/bench_rboehm_vec_of_copyable.rs

```diff
@@ -0,0 +1,20 @@
+//! ```cargo
```

Comment:
Does this comment tell cargo to use these features? I haven't seen this before!

Reply:
It's a `cargo-script` feature. It's really useful for these sorts of things because we don't have to compile all the dependencies manually.

Reply:
I must admit, I hadn't really appreciated what `cargo-script` did before now!